src/content/examples/CustomScrollbars/style.scss

```diff
@@ -1,32 +1,39 @@
 #example {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-
-  .scrollbars {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr 1fr;
-    align-items: center;
-    justify-items: center;
-    width: 400px;
-    height: 400px;
-
-    .scrollbar {
-      border: 2px dashed $color-gray;
-      border-radius: 4px;
-      width: 150px;
-      height: 150px;
-      font-size: 0.8em;
-
-      .scrollbar-inner {
-        padding: 4px 8px;
-      }
+  display: grid;
+  place-items: center;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  box-sizing: border-box;
+  padding: 20vh 30%;
```

Comment:
This makes them too far apart in my opinion:
![Screen Shot 2021-11-10 at 10 29 00](https://user-images.githubusercontent.com/7763083/141077645-032acdb9-00ba-473b-af28-e3de3f69a150.png)

Reply:
Ok I will make them closer